.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+#### Description of Change
+<!--
+Thank you for your Pull Request. Please provide a description above and review
+the requirements below.
+
+Pull Request process: https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process
+-->
+
+#### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] PR description included and stakeholders cc'd
+- [ ] `npm test` passes
+- [ ] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
+- [ ] manual tests are [changed or added](https://github.com/openfin/test_apps)
+- [ ] relevant documentation is [changed or added](https://github.com/hadoukenio/js-adapter)
```

Comment:
Does this signal the end of updating `javascript-adapter` docs?

Reply:
yea, we should be close to a public release of v2.